zhmccli/_cmd_partition.py

```diff
@@ -439,6 +441,42 @@ def partition_console(cmd_ctx, cpc, partition, **options):
                                                       options))
 
 
+@partition_group.command('mountiso', options_metavar=COMMAND_OPTIONS_METAVAR)
+@click.argument('CPC', type=str, metavar='CPC')
+@click.argument('PARTITION', type=str, metavar='PARTITION')
+@click.option('--imagefile', type=str, required=True,
+              help='The path and name of the ISO image.')
```

Comment:
Instead of "path and name of the ISO image", say "file path of the ISO image file".

Reply:
Done